casanovo/denovo/dataloaders.py

```
@@ -342,6 +355,288 @@ def _make_dataset(
+                                            ]
+                                        ):
+                                            prec_to_spec[(mz, rt, charge)][
+                                                "ms1_scans"
+                                            ] = []
+                                            prec_to_spec[(mz, rt, charge)][
+                                                "ms1_rts"
+                                            ] = []
+                                        prec_to_spec[(mz, rt, charge)][
+                                            "ms1_scans"
+                                        ].append(
+                                            [x for x in zip(mzs, intensities)]
```

Ruff (0.15.21)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)